patterns/2-structured/project-setup/templates/CODE_OF_CONDUCT-template.md

```diff
@@ -0,0 +1,78 @@
+<!--
+Variables to be replaced:
+- {{ Community-Name }}
+- {{ CommunityMail-Link }}
+-->
+
+# Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, {{ Community-Name }} as
```

Comment:
Will this sentence read correctly when the placeholder is replaced?

I tried this with the example of our own patterns community:

> In the interest of fostering an open and welcoming environment, the **InnerSource Patterns Community** as contributor and maintainer pledge to making participation in this project a harassment-free experience for everyone, ...

I think it is the "as contributor and maintainer" that is throwing me off a bit. Not quite sure.